### PR TITLE
[rfxcom] Updates / small fixes

### LINKED ISSUE #10251: [rfxcom] Energy Sensor - Exception while formatting value (d != java.math.BigDecimal)
Error with rfxcom Energy Sensor Channel, Exception while formatting value

My issue...
`[WARN ] [e.internal.SseItemStatesEventBuilder] - Exception while formatting value '7520157.67260111' of item HouseEnergyMonitor_TotalUsage with format '%d Wh': d != java.math.BigDecimal`

Belive it has something to do with the channels.xml for this energy sensor thing, having the state pattern for _totalusage_  as "%d Wh" but receiving a decimal number?
[https://github.com/openhab/openhab-addons/blob/main/bundles/org.openhab.binding.rfxcom/src/main/resources/OH-INF/thing/channels.xml](url)

image

* Version used: openHAB 3.0.0 Release Build, rfxcom 3.0.0
* Environment: openhabian on Raspberry Pi 3 Model B Rev 1.2 #

### PR DESCRIPTION
**Pull request by martinvw**
Added support for:
 - CHIME - 1byOne QH A19 rev10
 - CHIME - Alfawise, dBell
 - HUMIDITY - HUM3 - Inovalley S80 plant humidity sensor
 - LIGHTING1 - Oase Inscenio FM Master

Fixes #10251

### CODE REVIEW

File: bundles/org.openhab.binding.rfxcom/README.md [hunk 1]
```diff
@@ -609,6 +611,7 @@ A Humidity device
 
         *   HUM1 - LaCrosse TX3
         *   HUM2 - LaCrosse WS2300
+        *   HUM3 - Inovalley S80 plant humidity sensor
```
Comment (martinvw) [inline bundles/org.openhab.binding.rfxcom/README.md:614]: Please note that support for this was already in the code so no changes there

**Comment by lolodomo** [comment]
Please adjust the default translation properties file.

**Comment by martinvw** [comment]
@lolodomo done!

**Review by lolodomo — APPROVED** [review]
LGTM

**Pull request merged**